### ISSUE TITLE
Unknown table undefined error with double quotes in references

### ISSUE THREAD

**Issue by heuermh** [softwaretechnik-berlin/dbml-renderer#18]
We have spaces in table names, column names, and column types and as such have to use double quotes in references.  These are currently failing

```
$ cat foreign-keys.dbml
Table "Source" {
  "id" "int" [pk]
  "Target" "int" [ref: < "Target"."id"]
}
Table "Target" {
  "id" "int" [pk]
}

$ dbml-renderer -i foreign-keys.dbml -o foreign-keys.svg
Unknown table undefined
```

```  "Target" "int" [ref: < "Target".id]``` also fails

```  "Target" "int" [ref: < Target.id]``` succeeds

**Comment by heuermh** [comment]
I have found writing `Ref`s separately to be a workaround, double quotes are allowed in that part of the syntax.
```
$ cat seperate-foreign-keys.dbml
Table "Source" {
  "id" "int" [pk]
  "Target" "int"
}
Table "Target" {
  "id" "int" [pk]
}
Ref :"Source"."Target" < "Target"."id"

$ dbml-renderer -i seperate-foreign-keys.dbml -o seperate-foreign-keys.svg
```

**Comment by barnardb** [comment]
Thanks for the report. The fix is in v1.0.22.
